apps/notifications/src/app/templates/note-summary.tsx

```diff
@@ -0,0 +1,57 @@
+import {
+	__experimentalHStack as HStack,
+	__experimentalVStack as VStack,
+} from '@wordpress/components';
+import { html } from '../../panel/indices-to-html';
+import noticon2gridicon from '../../panel/utils/noticon2gridicon';
+import Gridicon from './gridicons';
+import ImagePreloader from './image-loader';
+import type { Note } from '../types';
+
+const NoteSummary = ( { note }: { note: Note } ) => {
+	return (
+		<HStack justify="flex-start" spacing={ 4 }>
+			<div style={ { position: 'relative', flexShrink: 0 } }>
+				<div style={ { width: '32px', height: '32px', borderRadius: '50%', overflow: 'hidden' } }>
+					<ImagePreloader
+						src={ note.icon }
+						key={ `image-preloader-${ note.icon }` }
+						placeholder={
+							// eslint-disable-next-line jsx-a11y/alt-text
+							<img src="https://www.gravatar.com/avatar/ad516503a11cd5ca435acc9bb6523536?s=128" />
+						}
+					/>
+				</div>
+				<span
+					className="wpnc__gridicon"
+					style={ {
+						position: 'absolute',
+						bottom: '-5px',
+						right: '-8px',
+						width: '16px',
+						height: '16px',
+						border: '1px solid #fff',
+						borderRadius: '50%',
+						background: '#ddd',
+					} }
+				>
+					<Gridicon icon={ noticon2gridicon( note.noticon ) } size={ 16 } />
```

Comment:
I believe we don't want to use gridicon anymore (switch to WPDS icon). So, we don't really need to clone the `gridicons.tsx` template file.

(We can follow up later)

Reply:
Yes. We should remove it later 🙂